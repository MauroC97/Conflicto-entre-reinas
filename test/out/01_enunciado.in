4 1 1 3 7 
2 7 10 
2 3 10 
3 4 4 9 
0 
0 
0 
0 
0 
0 
