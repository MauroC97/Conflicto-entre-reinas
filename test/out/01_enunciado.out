1 5 
1 6 
1 9 
1 9 
0 
0 
0 
0 
0 
0 
